frappe/desk/desktop.py

```diff
@@ -370,7 +370,7 @@ def get_workspace_sidebar_items():
 
 	filters = {
 		"restrict_to_domain": ["in", frappe.get_active_domains()],
-		"module": ["not in", blocked_modules],
+		"ifnull(module, '')": ("not in", blocked_modules)
```

Comment:
Can't reproduce. An empty list always adds `ifnull` for me. Try `run=0` to see generated query.